test/integrations/destinations/adobe_analytics/network.ts

```diff
@@ -0,0 +1,17 @@
+export const networkCallsData = [
+  {
+    httpReq: {
+      url: 'https://adobe.failure.omtrdc.net/b/ss//6',
+      params: {},
+      headers: {
+        'Content-type': 'application/xml',
+      },
+      method: 'POST',
+    },
+    httpRes: {
+      status: 400,
+      message:
+        '[ADOBE_ANALYTICS Response Handler] Request failed for destination adobe_analytics : NO pagename OR pageurl',
```

Comment:
The intention here is that `httpRes` is the raw axios response & not the response we see from network handler. Let me know if you need more clarification